source/user-manual/capabilities/active-response/ar-use-cases/wazuh-with-yara.rst

```diff
@@ -63,7 +63,9 @@ It defines the criteria used to execute a specific command:
 
 - The ``location`` setting as ``local``. It means that the active response is executed on the agent that generates the alert.
 
-- You can write a list of rule IDs that will trigger the active response in the ``rules_id`` setting. This example uses rule ``550``, new file added to the system, and rule ``554``, file modified in the system.
+- You can write a list of rule IDs that will trigger the active response in the ``rules_id`` setting. This example uses:
```

Comment:
I'm adding some content to the sentence so we can close it with a period. In this way, we can use ":" below in the listing and remove the commas which look a little awkward as a list. 
```suggestion
- You can write a list of rule IDs that will trigger the active response in the ``rules_id`` setting. This example uses the following rule IDs.
```